tools/storybook/.storybook/main.js

```diff
@@ -1,5 +1,18 @@
 const webpackOverride = require( '../webpack.config' );
 
+const staticDirs = [
+	{
+		from: '../../../plugins/woocommerce-admin/client',
+		to: 'main/plugins/woocommerce-admin/client',
+	},
+];
+if ( process.env.NODE_ENV && process.env.NODE_ENV === 'production' ) {
+	// Add WooCommerce Blocks Storybook for build process.
+	staticDirs.push( {
+		from: '../../../plugins/woocommerce-blocks/storybook/dist',
+		to: '/assets/woocommerce-blocks',
+	} );
+}
```

Comment:
Only add `woocommerce-blocks` stories when we are building storybook.
When building it live locally it makes more sense to run blocks storybook by itself.

Reply:
>  When building it live locally it makes more sense to run blocks storybook by itself.

Is this true?

Looking at it from the point of view of an external contributor, I don't think this does make sense. I'd expect all stories to be available with a single command if I was working on something and wanted to include my changes in Storybook, having them figure this difference out adds a bit of friction.

This would have to be clearly documented if we want to keep it this way.